spyder/app/tests/test_mainwindow.py

```diff
@@ -1436,6 +1436,28 @@ def test_run_cython_code(main_window, qtbot):
     main_window.editor.close_file()
 
 
+@pytest.mark.slow
+def test_project_path(main_window, tmpdir, qtbot):
+    """Test project path added to spyder_pythonpath."""
+    # Wait until the window is fully up
+    shell = main_window.ipyconsole.get_current_shellwidget()
+    qtbot.waitUntil(
+        lambda: shell._prompt_html is not None, timeout=SHELL_TIMEOUT)
+
+    projects = main_window.projects
+
+    # Create a project
+    path = to_text_string(tmpdir.mkdir('project_path'))
```

Comment:
```suggestion
    path = str(tmpdir.mkdir('project_path'))
```

`to_text_string` was a `str` version compatible with Python 2, but it's not necessary anymore.